packages/core/src/routes/session.ts

```diff
@@ -198,6 +199,12 @@ export default function sessionRoutes<T extends AnonymousRouter>(router: T, prov
         })
       );
 
+      if (!password) {
```

Comment:
1. the change does not match the title?
2. why do we need to remove password checking?

Reply:
can find more context at LOG-1468.
we already have username existence check when registering with username and password (username and password are provided at the same time). The current case is to do username existence check when a user finished entering his desired 'username' (password is not entered at the time). can discuss this later on weekly.

Reply:
issue description and implementation updated.